src/app/(admin)/new_dashboard/components/ChatHeader/types/index.ts

```diff
@@ -0,0 +1,9 @@
+export type Chat = {
+  messages: Message[];
+  user: User;
+};
+export interface User {
```

Comment:
we already have a user type in the frontend, we dont need to repeat on the same type twice